### ISSUE TITLE
0.11 still has non-free icons

### ISSUE THREAD

**Issue by luke-jr** [bitcoin/bitcoin#6367]
@jonasschnelli Poke.

**Comment by Diapolo** [comment]
You could perhaps help by listing which ones :).

**Comment by luke-jr** [comment]
assets-attribution.md still is unclear about which ones... :/

**Comment by jonasschnelli** [comment]
The list is there: https://github.com/bitcoin/bitcoin/blob/master/doc/assets-attribution.md#assets-used.
I have a look at this. First i try to get permissions to use the icons under the MIT license by contacting (ring up) Stephen Hutchings. If i don't succeed, i'll promise to adapt the icons by redrawing them.

**Comment by luke-jr** [comment]
I already contacted him (and all other contributors to typicons) 3 months ago, and got no answer from any of them. Better to not hold up 0.11, and just draw new ones (be sure they aren't inspired by typicons please; copyright doesn't care if you redraw!)

```
From: Luke Dashjr <luke@dashjr.org>
To: stephenhutchings <stephen@creativelicence.com.au>,
 Alexander Synaptic <alexander@synapticism.com>,
 Aleksey V Zapparov <ixti@member.fsf.org>,
 Vitaly Puzrin <vitaly@rcdesign.ru>,
 Richard Bailey <additive.inverse@gmail.com>
Subject: typicons licensing
Date: Tue, 7 Apr 2015 11:00:53 +0000
Message-Id: <201504071100.56262.luke@dashjr.org>

Hi,

A contributor to Bitcoin Core has included modified versions of some of your 
typicons in his changes. However, for consistency/clarity the project would 
like to keep everything under a common MIT license:
    http://opensource.org/licenses/MIT

Would you be willing to allow these MIT license terms as an alternative to the 
CC BY SA license you presently offer for typicons? If not, please let me know 
that as well, so we can find replacements.

The specific images involved can be seen at:
    https://github.com/bitcoin/bitcoin/commit/31aac02446472ec5bfc4676ab190ec9d37056503#diff-2f5955e1426db3870f0f22b667007379

For disclosure, I am also concerned that section 4(d) of the CC BY SA license 
is not in the spirit of free/open licensing, and would present problems should 
people producing derived works of Bitcoin Core wish to use the GPL or similar 
copyleft licenses.

Thanks,

Luke
```

**Comment by luke-jr** [comment]
@stephenhutchings @synapticism @additiveinverse @puzrin @ixti One last attempt to reach you (this time via GitHub)... please respond ASAP.

**Comment by stephenhutchings** [comment]
Hi Luke, happy for these to be distributed under the terms of the MIT licence. Let me know if you need anything further from me.

**Comment by luke-jr** [comment]
Pleasant surprise, thanks @stephenhutchings !

**Comment by jonasschnelli** [comment]
Nice! Many thanks @stephenhutchings.

**Comment by laanwj** [comment]
Many thanks @stephenhutchings . License change has been merged per #6369.
